scripts/pre_push_hook.py

```diff
@@ -37,8 +37,11 @@
 import subprocess
 import sys
 
-# pylint: enable=wrong-import-order
+# pylint: disable=relative-import
+import build
```

Comment:
Can we do `from . import xyz` and remove `pylint disable...`?

Reply:
I tried that. It gives an error:

```
Traceback (most recent call last):
  File ".git/hooks/pre-push", line 40, in <module>
    from . import build
ValueError: Attempted relative import in non-package
```

Reply:
Thanks for checking!